api/URL.json

```diff
@@ -960,10 +928,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL/toJSON",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
Chrome 71. https://storage.googleapis.com/chromium-find-releases-static/95c.html#95c28adc1d9b521918df000afb77e2190ca6dbd5

Android and Webview also.